### PR TITLE
new subnet design

### LINKED ISSUE #393: new subnet design
as a platform admin, I need the client network to include only the following subnets in nane1 and nane2
unclassified - main (for GCE)
protected - main (for GCE)
others - regional managed proxy
others - private service connect

regional managed proxy is required for secure web proxy
private service connect is for access to common services that are offered to all clients

### PR DESCRIPTION
**Pull request by davelanglois-ssc**
The following design changes are implemented based on a decision that was taken during a meeting with SSC network engineering.
- removed protected a subnets. 
- changed subnets names to nonp and pbmm
- adjusted hierarchichal firewall policies
- fixes minor issue with the client-project-setup package where setters were causing some confusion.

closes #393

### CODE REVIEW

**Review by cartyc — APPROVED** [review]

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

**Review by fmichaelobrien — APPROVED** [review]
LGTM
deploying clz in separate org

File: solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/subnet.yaml [hunk 1]
```diff
@@ -12,25 +12,25 @@
...
 # - logging enabled for flow logs https://cloud.google.com/vpc/docs/using-flow-logs
 # - private google access enabled https://cloud.google.com/vpc/docs/private-google-access
 #########
-# Subnet unclassified northamerica-northeast1
+# Subnet nonp-main northamerica-northeast1
 # AC-4, AC-4(21)
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeSubnetwork
 metadata:
-  name: host-project-id-nane1-standard-unclassified-snet # kpt-set: ${host-project-id}-nane1-standard-unclassified-snet
```
Comment (alaincormier-ssc) [inline solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/subnet.yaml:24]: not a show stopper, but worth mentioning that the 'unclassified' subnet is used in this example:

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/55301f0804c02cc177c01b2048e12e646d66c96e/examples/landing-zone-v2/tier4/managed-instance-group/instance-template.yaml#L58

File: solutions/client-landing-zone/README.md [hunk 2]
```diff
@@ -144,7 +138,7 @@ This package has no sub-packages.
 
 1.  Move into the local package:
     ```shell
-    cd "./solutions/client-landing-zone/"
+    cd "./client-landing-zone/"
```
Comment (alaincormier-ssc) [inline solutions/client-landing-zone/README.md:141]: I noticed earlier this week that the new generate-docs script removes the 'solutions' folder in this readme's step

**Review by alaincormier-ssc — APPROVED** [review]

File: solutions/client-landing-zone/README.md (hunk 2 above)
Comment (davelanglois-ssc) [reply solutions/client-landing-zone/README.md:141]: yeah and that is on purpose. after executing the kpt pkg get command,, it is really that folder that is created.

File: solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/subnet.yaml (hunk 1 above)
Comment (davelanglois-ssc) [reply solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/subnet.yaml:24]: fixed

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Review by fmichaelobrien — APPROVED** [review]
LGTM
Last change - unclassified to nonp reviewed

**Review by alaincormier-ssc — DISMISSED** [review]

**Review by alaincormier-ssc — APPROVED** [review]

**Review by fmichaelobrien — DISMISSED** [review]
LGTM
- reviewed last dns changes

**Comment by davelanglois-ssc** [comment]
@lucstjean-ssc has an important fix to the dns feature that will be implemented prior to this PR. I will merge main afterwards

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

**Review by alaincormier-ssc — DISMISSED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM
client lz sec controls diff looks good

**Review by cartyc — DISMISSED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM
- name changes

File: solutions/client-project-setup/setters.yaml [hunk 3]
```diff
@@ -66,8 +66,8 @@ data:
   host-project-id: net-host-project-12345
   #
   # the nane1 and nane2 subnets to grant network user permissions, replace 'classification' with appropriate values
-  allowed-nane1-subnet: nane1-standard-classification-snet
-  allowed-nane2-subnet: nane2-standard-classification-snet
+  allowed-nane1-main-subnet: host-project-id-nane1-standard-classification-main-snet
```
Comment (davelanglois-ssc) [inline solutions/client-project-setup/setters.yaml:69]: @borkodjurkovic-ssc this is to fix the issue you showed me this morning

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Review by alaincormier-ssc — APPROVED** [review]

**Pull request merged**